app/search/search.component.html

```diff
@@ -11,10 +11,13 @@
                (keyup)="search(searchBox.value)"
                [value]="term"
                />
+        <span class="input-group-addon downloader">
+            <img class="downloader" (click)="download(term)" src='../../assets/img/download-csv.svg' />        
```

Comment:
I think it's better to include images from css, like this - 
https://github.com/OpenBudget/budgetkey-app-search/blob/master/app/styles.scss#L637

if you have to do it from the img tag, then I would check if referring to the img with ../../ is right, I think it's dependant on the URL of the page, which might be problematic. Worth to check what's the angular way of doing it (I'm not sure myself..)

Reply:
i'll check it out